armi/reactor/tests/test_reactors.py

```diff
@@ -211,6 +212,7 @@ def loadTestReactor(
         # protocol=2 allows for classes with __slots__ but not __getstate__ to be pickled
         TEST_REACTOR = cPickle.dumps((o, o.r, assemblies.getAssemNum()), protocol=2)
 
+    o.r.sort()
```

Comment:
Is this needed? `r.sort()` gets called in the `isPickledReaactor` above and via `reactors.loadFromCs(cs)` so I'm not sure that it is?

Reply:
I believe it _is_ necessary, yes.  Because just above, we are manually adding assemblies one-at-a-time to the `Reactor`:

https://github.com/terrapower/armi/blob/996341f33f9629796e959f7110e8a676d31eb648/armi/reactor/tests/test_reactors.py#L204-L206

And those are not necessarily added in their geometric order, right?

Reply:
That's true. They are going into the spent fuel pool though, not the core. If we're interested in ordering the sfp only, should/could the call be `o.r.core.sfp.sort()`? That way we aren't sorting through the already sorted core too (if I understand right).

Reply:
I understand it's not desperately important, but this test reactor is used in a LOT of places, and I want to keep the entire `Reactor` sorted.  I mean, "it's just the Spent Fuel Pool" is probably fair, but somewhere out there, someone cares about the SFP.